test/src/resolvers/testResolver/index.spec.ts

```diff
@@ -1,4 +1,4 @@
-import { BindingTypes, testBindings } from '../../../common/bindingTester';
+import { BindingTypes, testBindings } from '../../../common/testers/bindingTester';
 import { ITestResolver, testResolverModulesBinder } from '../../../../src/resolvers/testResolver';
 import { TestResolver } from '../../../../src/resolvers/testResolver/impl/testResolver';
 
```

Comment:
there is only import and no export here..

Reply:
? this is a test which invokes the testBindings method. No need to export anything.
Am I missing something?

Reply:
so there is no invokation what so ever?

Reply:
there is one just 1 line below